src/client/components/household-information/AnnualIncomeSection.jsx

```diff
@@ -43,15 +43,15 @@ class AnnualIncomeSection extends React.Component {
             <table className="review usa-table-borderless">
               <tbody>
                 <tr>
-                  <td>Children Gross Income:</td>
+                  <td>Gross Annual Income from Employment:</td>
```

Comment:
Yesterday we said that we were going to leave the section titles "Veteran", "Spouse", "Child: Name" and all of the input labels would be the same.